packages/app/android/build.gradle

```diff
@@ -105,6 +106,22 @@ dependencies {
   implementation "com.google.android.gms:play-services-auth:${ReactNative.ext.getVersion("play", "play-services-auth")}"
 }
 
+def jvmVersion = Jvm.current().javaVersion?.majorVersion
+// if it cannot get version, just allow it to pass. This is just a helper for developers
+if ((jvmVersion?.toInteger() ?: 17) < 17) {
```

Comment:
and this will make it so that even people who don't read docs will fail fast :-), big +1 for me
I think there are people *still* trying to build with JDK 11 but I honestly don't know they are making it work and I tried - I can usually make it work 🤷 . 17 should work for everyone though, and really we should all be on 21